kotlin/Dynamic Programming/LongestPalindromInAString.kt

```diff
@@ -0,0 +1,38 @@
+fun longestPalindromInAString(s: String): String {
```

Comment:
Please fix typo in file name and function

Reply:
Updated by this commit - https://github.com/ByteByteGoHq/coding-interview-patterns/pull/43/commits/b306f226e0d2113c25be443239d0976612cd39e1